package.json

```diff
@@ -48,7 +48,7 @@
     "react": "^15.4.2",
     "react-dom": "^15.4.2",
     "webpack": "^3.6.0",
-    "webpack-dev-server": "^2.9.1",
+    "webpack-dev-server": "2.7.1",
```

Comment:
I don't think we should do this.. Example my team isn't locked down to IE10 and we can upgrade. The peer dependency should be flexible

Reply:
@mhemesath Updated.